web/swagger-docs/changelog.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## [2.2.2] & [1.3.2] 2020-10-27
+
+### Changed
+* __OrderPayment__._provider_ and __OrderPayment__._method_ are now deprecated and will be removed on the 1st of February 2021. These fields have been replaced with __OrderPayment__._gateway_name_ and __OrderPayment__._method_name_ respectively.  
+
```

Comment:
The new payment gateways endpoints may be added, and the old paytypes endpoints may be listed as deprecated (also scheduled to be removed on feb 1).

Reply:
Added those as well (accidentally squashed the fixup commit though)